cli/test/lib/exec/spawn_spec.js

```diff
@@ -171,6 +173,20 @@ describe('lib/exec/spawn', function () {
       })
     })
 
+    context('detects kill signal', function () {
+      it('exits with error on SIGKILL', function () {
+        this.spawnedProcess.on.withArgs('exit').yieldsAsync(null, 'SIGKILL')
+
+        return spawn.start('--foo')
+        .then(() => {
+          throw new Error('should have hit error handler but did not')
+        }, (e) => {
+          debug('error message', e.message)
+          snapshot(e.message)
+        })
+      })
+    })
+
```

Comment:
@bahmutov I think it would be worth it to add an `e2e` test for this, since it involves multiple processes, what do you think?

Reply:
would be tricky though, maybe not worth the time investment unless we have issues with this PR going forward

Reply:
I am not so sure e2e would be worth it
- we don't have e2e tests for CLI right now
- Node side (that one that delivers `code: null, signal: <name>` is pretty solid :)
- we have only changed our handling, which is covered by the tests
- finally, so far this has affected only very small number of our users I believe, otherwise we would hear about it again and again